LICENSE.md

```diff
@@ -1,11 +1,11 @@
-All components of this product are Copyright (c) 2011-2023 Phrase GmbH.  All rights reserved.
+All components of this product are Copyright (c) 2011-2025 Phrase GmbH. All rights reserved.
```

Comment:
shouldn't this one then be the same as https://github.com/phrase/ios-sdk-src/pull/103/files ?
Because right now they differ.